web/yo/app/views/drugs.html

```diff
@@ -0,0 +1,49 @@
+<div class="drugs">
+        <h3 class="headerStyle">Curated Drugs</h3>
+        <div ng-if="showTable==true">
+            <table class="table" datatable="ng" dt-options="dtOptions" dt-columns="dtColumns">
+                <thead>
+                <tr class="dashedHeader">
+                    <th>Name</th>
+                    <th>NCI Thesaurus Code</th>
+                    <th>Description</th>
+                </tr>
+                </thead>
+                <tbody>
+                <tr data-ng-repeat="drug in drugList">
+                    <td>
+                        <div ng-if="drug.editDrugName==false">
```

Comment:
`editDrugName` really don't have to be initialized. It could just live at the ng-repeat cycle. Let's talk about this.